src/ORM/Marshaller.php

```diff
@@ -133,7 +138,7 @@ public function one(array $data, array $options = [])
             $columnType = $schema->columnType($key);
             if (isset($propertyMap[$key])) {
                 $assoc = $propertyMap[$key]['association'];
-                $value = $this->_marshalAssociation($assoc, $value, $propertyMap[$key]);
+                $value = $this->_marshalAssociation($assoc, $value, $propertyMap[$key] + $marshallOptions);
```

Comment:
Here is a test:

https://gist.github.com/Tzaoh/23b01926b3aa773b41ca

As you can see, if you specify a non-existent primaryKey the marshaller told that is new (and the record would be created) so it would not be necesary a logic with the `forceNew` setting IMHO.